plugin-development/visual-studio-code-extension.mdx

```diff
@@ -0,0 +1,78 @@
+---
+title: 'Amplication Visual Studio Code Extension'
+description: 'Learn how to install and use the Amplication VS Code extension to streamline plugin development.'
+sidebarTitle: 'VS Code Extension'
+icon: 'keyboard'
+---
+
+The Amplication VS Code extension provides a streamlined way to develop and manage plugins for Amplication directly from your IDE. This extension offers features to download plugin templates, manage dependencies, build, push, and publish new plugin versions effortlessly.
+
+## Features
+
+- **Download Plugin Templates**: Select and download pre-existing templates from Amplication’s GitHub repositories.
+- **Manage Dependencies**: Automatically install dependencies for downloaded templates.
+- **Build and Push**: Build your plugin and push updates to your repository.
+- **Version Publishing**: Automate version bumping and publishing directly from VS Code.
+- **Seamless Integration**: Works directly with Amplication’s plugin development workflow.
+
+## Installation
+
+To install the Amplication VS Code extension:
+
+1. Open VS Code.
+2. Go to the Extensions Marketplace (`Ctrl+Shift+X` or `Cmd+Shift+X` on macOS).
+3. Search for `Amplication`.
+4. Click **Install**.
+
+Alternatively, install it via the command line:
+
+```sh
+code --install-extension amplication.amplication
+```
+
+## Usage
+
+The commands are accessible from the **command palette** (Ctrl+Shift+P or Cmd+Shift+P on macOS).
+
+<Steps>
+
+<Step title="Download a Plugin Template">
+```
+Amplication Plugin: Download Blueprint Template Plugin
+```
+Select the desired template from the available options. Currently available .Net, Node.js and Blueprint.
+</Step>
+
+</Steps>
+
+<Step title="Install Dependencies">
+```
+Amplication Plugin: Install Dependencies
+```
+After downloading a template plugin, you should run this command to install the plugin's dependencies, specified in the package.json file within the plugin's directory.
```

Comment:
This can be done for any plugin, regardless if downloaded or not

Reply:
I wanted to create a flow, but, ok